csrc/quantization/smoothquant/quant_utils.cuh

```diff
@@ -0,0 +1,243 @@
+// Adated from FasterTransformer, https://github.com/NVIDIA/FasterTransformer/blob/release/v5.3_tag/src/fastertransformer/kernels/decoder_masked_multihead_attention/decoder_masked_multihead_attention_template.hpp
```

Comment:
@varun-sundar-rabindranath 

can you audit which of these is needed?

Reply:
Yes - we needed only function from this file. Copied it to the kernel file and removed it from here. (https://github.com/neuralmagic/nm-vllm/pull/215)